docs/examples/indie-linter-v2.md

```diff
@@ -0,0 +1,95 @@
+# Indie Linter v2
+
+The [Indie Linter API v2](../types/indie-linter-v2.md) demoed below supports [Messages v2](../types/linter-message-v2.md) only.
```

Comment:
This should probably have some examples of where the Indie API might be useful to use.